pkg/component/certmanagement/crd.go

```diff
@@ -23,26 +25,41 @@ var (
 	//go:embed assets/crd-cert.gardener.cloud_issuers.yaml
 	crdIssuers string
 
-	resources []string
+	// resourceObjectKeyMap maps the ObjectKey of the CRD to its corresponding manifest
+	resourceObjectKeyMap map[client.ObjectKey]string
```

Comment:
```suggestion
	crdNameToResource map[string]string
```

Reply:
The name suggestion sounds good, but I still would prefer the `client.ObjectKey` as key for the map.